forc-plugins/forc-crypto/src/lib.rs

```diff
@@ -0,0 +1,36 @@
+pub mod address;
+pub mod args;
+pub mod keccak256;
+pub mod keys;
+pub mod sha256;
+
+pub(crate) fn help() -> &'static str {
```

Comment:
The other forc plugins in this repo use the `forc_util::cli_examples` macro. It might be good to use that here for consistency.

Reply:
cannot find an example specifically for top-level help; also this code remains unchanged as it was migrated from `main.rs`.